### ISSUE TITLE
Real-time daemon seems to ingore --keep-last

### ISSUE THREAD

**Issue by fancsali** [allinurl/goaccess#1951]
I am running the following command as a `systemd` service on my server:

```
goaccess /var/log/apache2/access.log.1 /var/log/apache2/access.log --restore --persist --db-path /var/spool/goaccess/realtime --real-time-html --keep-last=10 -o /var/lib/goaccess/realtime.html --daemonize --pid-file=/var/run/goaccess.pid
```

However, it seems, that it does ignore the 10-day window altogether:

<img width="1127" alt="image" src="https://user-images.githubusercontent.com/1670136/97718202-9243f380-1abd-11eb-96a6-753bd0beb27f.png">

**Comment by allinurl** [comment]
There were some fixes on `--keep-last` upstream. Could you please build from [**development**](https://github.com/allinurl/goaccess#build-from-github-development) and let me know if that solves the issue on your end.

Please note that you will need to either delete your current database files or change --db-path to point to a new set of db files, otherwise it will restore everything is on disk.

**Comment by allinurl** [comment]
Thanks for reporting it. 

I'm closing this as I believe it was fixed in v1.4.1. Feel free to reopen it if still not working for you in v1.4.1.
